chameleon/src/test/java/org/hibernate/omm/AbstractMongodbContainerTests.java

```diff
@@ -22,8 +22,7 @@ protected SessionFactory getSessionFactory() {
 					"org.hibernate.omm.jdbc.MongodbConnectionProvider"
 			);
 			cfg.setProperty(
-					MongodbAvailableSettings.MONGODB_CONNECTION_URL,
-					"mongodb+srv://nathanqingyangxu:I0Oj6kwaf3r1ZuRh@cluster0.gfxzieb.mongodb.net/?retryWrites=true&w=majority&appName=Cluster0"
+					MongodbAvailableSettings.MONGODB_CONNECTION_URL, "mongodb://localhost/?directConnection=false"
```

Comment:
Best to have the connection string that's checked in to the repo refer to a local deployment

Reply:
I am torn apart on how to go about Mongodb integration testing. There are multiple options:

- embedded mongodb as in OGM; 
- docker based opencontainer + Mongodb docker image
- github action as in Hibernate ORM (benefit is to run multiple mongodb versions at the same time)

At current initial stage, I am not confident about the above three and I prefer inspecting mongodb web console whenever possible to ensure my code is working, so I delayed choosing one of the above options.

Reply:
I create a hibernate.properties file with mongodb configs as place holders at https://github.com/NathanQingyangXu/JPA-MongoDB-mapping/blob/main/chameleon/src/test/resources/hibernate.properties

In the future a good integration testing plan will be chosen as planned above.